sats-dna/src/main/kotlin/com/sats/dna/components/SatsTag.kt

```diff
@@ -21,19 +21,29 @@ fun SatsTag(
         SatsTagColor.Primary -> SatsTheme.colors2.graphicalElements.tags.primary.bg
         SatsTagColor.Secondary -> SatsTheme.colors2.graphicalElements.tags.secondary.bg
         SatsTagColor.Featured -> SatsTheme.colors2.graphicalElements.tags.featured.bg
+        SatsTagColor.Pt -> SatsTheme.colors2.graphicalElements.workouts.pt.bg
+        SatsTagColor.Gx -> SatsTheme.colors2.graphicalElements.workouts.gx.bg
+        SatsTagColor.Gymfloor -> SatsTheme.colors2.graphicalElements.workouts.gymfloor.bg
+        SatsTagColor.OwnTrainingOther ->
+            SatsTheme.colors2.graphicalElements.workouts.ownTrainingOther.bg
     }
 
     val contentColor = when (color) {
         SatsTagColor.Primary -> SatsTheme.colors2.graphicalElements.tags.primary.fg
         SatsTagColor.Secondary -> SatsTheme.colors2.graphicalElements.tags.secondary.fg
         SatsTagColor.Featured -> SatsTheme.colors2.graphicalElements.tags.featured.fg
+        SatsTagColor.Pt -> SatsTheme.colors2.graphicalElements.workouts.pt.bg
+        SatsTagColor.Gx -> SatsTheme.colors2.graphicalElements.workouts.gx.bg
+        SatsTagColor.Gymfloor -> SatsTheme.colors2.graphicalElements.workouts.gymfloor.bg
+        SatsTagColor.OwnTrainingOther ->
+            SatsTheme.colors2.graphicalElements.workouts.ownTrainingOther.bg
     }
 
     SatsTagLayout(text, backgroundColor = backgroundColor, contentColor = contentColor, modifier)
 }
 
 enum class SatsTagColor {
-    Primary, Secondary, Featured,
+    Primary, Secondary, Featured, Pt, Gx, Gymfloor, OwnTrainingOther,
 }
```

Comment:
[I don't see these tags in the Figma component.](https://www.figma.com/file/Nu1V7557V7KovgpZCNDObQ/%F0%9F%93%B1-sats-app-components-new?type=design&node-id=2109-32165&mode=design) Could we push design to add them there? I'm wondering it it makes sense to make a separate `WorkoutTag` component instead, similar to how we have the `RewardsTag`, but that kind of depends on how they set it up in the design. Right now, the component's rules state:

> Should only be used in these three colors, depending on surface and circumstances. The only exception being the “rewards labels” that are understood as a different component.

Reply:
I got design to add the workout tag and updated the PR to implement your suggestions